### PR TITLE
Account management updates

### PR DESCRIPTION
**Pull request by pzaichkina**
# Description

Updates to the account management page - includes account settings and new option to change passwords.

## Type of change

Please delete options that are not relevant.

- [X] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/other/portal/account-management.md [hunk 1]
```diff
@@ -1,20 +1,37 @@
...
-Use your verified registration email and password to access the Codat portal and see your clients' data. This is applicable to the UAT and production environments.
+If you want to change your password (for example, for security reasons), you can do so on the [Account settings](https://app.codat.io/settings/account) page:
 
-You can change your password for security reasons, or reset it if you forgot it. To change your password, perform the following steps:
+1. In the **Password** section, enter your current password.
+2. Then, enter and confirm your new password. We provide you with a list of password requirements, and measure the strength of your new password.
+3. Finally, press **Update password** to save your changes. 
 
-1. Click **Forgot your password?** on the sign-in screen.
+If you forgot your password and are looking to reset it, perform the following steps:
```
Comment (mcclowes) [inline docs/other/portal/account-management.md:26]:
```suggestion
If you have forgotten your password and are looking to reset it:
```

File: static/img/other-guides/0012-acct-mgt-access.png
Comment (mcclowes) [inline static/img/other-guides/0012-acct-mgt-access.png:1]: Looks weird with the browser bar. You can disable this

**Review by mcclowes — CHANGES_REQUESTED** [review]

File: docs/other/portal/account-management.md [hunk 2]
```diff
@@ -1,20 +1,37 @@
...
+description: "Learn how to change your Codat account settings and reset your password"
 ---
 
+## Managing your account
+
+You can now change details and settings of your user profile in the [Codat Portal](https://app.codat.io/). Click on a circle with your initials in the top right corner of the portal to access the [Account settings](https://app.codat.io/settings/account) page. 
+
+![A screenshot of the dropdown menu that allows to navigate to account settings](/img/other-guides/0012-acct-mgt-access.png)
+
+In the **Profile** section, you can change the name associated with the account, and confirm your involvement with the technical implementation of your Codat solution. Remember to press **Save changes** when you are done.
```
Comment (d-laing) [inline docs/other/portal/account-management.md:12]:
```suggestion
In the **Profile** section, you can change the name associated with the account, and confirm your involvement with the technical implementation of your Codat solution. Remember to click **Save changes** when you are done. 
```

File: docs/other/portal/account-management.md [hunk 3]
```diff
@@ -1,20 +1,37 @@
...
+
 ## Changing your password
 
-Use your verified registration email and password to access the Codat portal and see your clients' data. This is applicable to the UAT and production environments.
+If you want to change your password (for example, for security reasons), you can do so on the [Account settings](https://app.codat.io/settings/account) page:
 
-You can change your password for security reasons, or reset it if you forgot it. To change your password, perform the following steps:
+1. In the **Password** section, enter your current password.
+2. Then, enter and confirm your new password. We provide you with a list of password requirements, and measure the strength of your new password.
+3. Finally, press **Update password** to save your changes.
```
Comment (d-laing) [inline docs/other/portal/account-management.md:24]:
```suggestion
3. Finally, click **Update password** to save your changes. 
```

Comment (d-laing) [reply docs/other/portal/account-management.md:24]: **Update password** is styled as a button, so I think we should use "click".

File: docs/other/portal/account-management.md [hunk 4]
```diff
@@ -1,20 +1,37 @@
...
-You can change your password for security reasons, or reset it if you forgot it. To change your password, perform the following steps:
+1. In the **Password** section, enter your current password.
+2. Then, enter and confirm your new password. We provide you with a list of password requirements, and measure the strength of your new password.
+3. Finally, press **Update password** to save your changes. 
 
-1. Click **Forgot your password?** on the sign-in screen.
+If you have forgotten your password and are looking to reset it:
 
+1. Click **Forgot your password?** on the sign-in screen.
 2. Next, enter the email you registered with.
```
Comment (d-laing) [inline docs/other/portal/account-management.md:29]: I don't think this short procedure needs "next" and "finally". I appreciate this text was already there!

**Review by d-laing — CHANGES_REQUESTED** [review]
Clear and concise.

**Review by mcclowes — APPROVED** [review]

**Pull request merged**